daphne_worker_test/helper.env

```diff
@@ -15,7 +15,7 @@ DAP_HELPER_STATE_STORE_GARBAGE_COLLECT_AFTER_SECS = '10'
 #
 # NOTE(nakatsuka-y) The leader_url and helper_url must end with a "/".
 # When adding paths, they must not start with a "/".
-DAP_TASK_LIST = '{"f285be3caf948fcfc36b7d32181c14db95c55f04f55a2db2ee439c5879264e1f":{"version":"v02","leader_url":"http://leader:8787/v02/","helper_url":"http://helper:8788/v02/","collector_hpke_config":{"id":23,"kem_id":"X25519HkdfSha256","kdf_id":"HkdfSha256","aead_id":"Aes128Gcm","public_key":"ec6427a49c8e9245307cc757dbdcf5d287c7a74075141af9fa566c293a52ee7c"},"time_precision":3600,"query":{"time_interval":{"min_batch_size":10}},"vdaf":{"prio3":{"sum":{"bits":10}}},"vdaf_verify_key":"1fd8d30dc0e0b7ac81f0050fcab0782d"},"6d2ef497d369fd2e72d96c47e5c28496800f5e546e912554d0df83bf24a25378":{"version":"v02","leader_url":"http://leader:8787/v02/","helper_url":"http://helper:8788/v02/","collector_hpke_config":{"id":23,"kem_id":"X25519HkdfSha256","kdf_id":"HkdfSha256","aead_id":"Aes128Gcm","public_key":"ec6427a49c8e9245307cc757dbdcf5d287c7a74075141af9fa566c293a52ee7c"},"time_precision":3600,"query":{"fixed_size":{"min_batch_size":10,"max_batch_size":12}},"vdaf":{"prio3":{"sum":{"bits":10}}},"vdaf_verify_key":"d4c0fd634e80005dc514eb4c9538fbf7"}}'
+DAP_TASK_LIST = '{"f285be3caf948fcfc36b7d32181c14db95c55f04f55a2db2ee439c5879264e1f":{"version":"v02","leader_url":"http://leader:8787/v02/","helper_url":"http://helper:8788/v02/","collector_hpke_config":{"id":23,"kem_id":"X25519HkdfSha256","kdf_id":"HkdfSha256","aead_id":"Aes128Gcm","public_key":"ec6427a49c8e9245307cc757dbdcf5d287c7a74075141af9fa566c293a52ee7c"},"time_precision":3600,"expiration": 30695925405,"query":{"time_interval":{"min_batch_size":10}},"vdaf":{"prio3":{"sum":{"bits":10}}},"vdaf_verify_key":"1fd8d30dc0e0b7ac81f0050fcab0782d"},"6d2ef497d369fd2e72d96c47e5c28496800f5e546e912554d0df83bf24a25378":{"version":"v02","leader_url":"http://leader:8787/v02/","helper_url":"http://helper:8788/v02/","collector_hpke_config":{"id":23,"kem_id":"X25519HkdfSha256","kdf_id":"HkdfSha256","aead_id":"Aes128Gcm","public_key":"ec6427a49c8e9245307cc757dbdcf5d287c7a74075141af9fa566c293a52ee7c"},"time_precision":3600,"expiration": 30695925405,"query":{"fixed_size":{"min_batch_size":10,"max_batch_size":12}},"vdaf":{"prio3":{"sum":{"bits":10}}},"vdaf_verify_key":"d4c0fd634e80005dc514eb4c9538fbf7"}}'
```

Comment:
Reviewer note: Expiration dates for tasks are set to 1000 years from today. This is a hack: Once we implement #138 , /internal/test/add_task, we'll be able to replace these hardcoded configs with configs generated on the fly.